config/settings.yml

```diff
@@ -83,6 +83,9 @@ features:
     # actually starting when it would be set to false
     has_current_cycle_started?: true
     show_next_cycle_allocation_recruitment_page: false
+  maintenance_mode:
+    enabled: false
+    message: "This service will be unavailable from Thursday 4 March at 17:00 to Friday 5 March at 09:00, while we carry out maintenance.\n\nFind teacher training courses will still be available during this time. If you have any questions, contact us at becomingateacher@digital.education.gov.uk"
```

Comment:
Can we use translation en.yml file. 
https://guides.rubyonrails.org/i18n.html#using-safe-html-translations
Pretty please.

Reply:
Yep, makes sense.

Reply:
Discussed this offline. We're going to leave in here so as we can update the message from ENV vars if need be without deploying.